src/custom/state/claim/hooks/index.ts

```diff
@@ -44,20 +50,61 @@ import {
   setSelectedAll,
   ClaimStatus,
 } from '../actions'
+import { EnhancedUserClaimData } from 'pages/Claim/types'
+import { supportedChainId } from 'utils/supportedChainId'
 
 const CLAIMS_REPO_BRANCH = 'main'
 export const CLAIMS_REPO = `https://raw.githubusercontent.com/gnosis/cow-merkle-drop/${CLAIMS_REPO_BRANCH}/`
 
+// Base amount = 1 VCOW
+const ONE_VCOW = CurrencyAmount.fromRawAmount(
+  V_COW[SupportedChainId.RINKEBY],
+  parseUnits('1', V_COW[SupportedChainId.RINKEBY].decimals).toString()
+)
 // TODO: these values came from the test contract, might be different on real deployment
+// TODO: check the invert call here, probably not necessary when we get actual numbers
+// Inverted price because using VCOW as base
 // Network variable price
-export const NATIVE_TOKEN_PRICE = {
-  [SupportedChainId.MAINNET]: '37500000000000', // '0.0000375' WETH (18 decimals) per vCOW, in wei
-  [SupportedChainId.RINKEBY]: '37500000000000', // assuming Rinkeby has same price as Mainnet
-  [SupportedChainId.XDAI]: '150000000000000000', // TODO: wild guess, wxDAI is same price as USDC
+export const NATIVE_TOKEN_PRICE: { [chain in SupportedChainId]: Price<Currency, Currency> } = {
+  // '0.0000375' WETH (18 decimals) per vCOW, in wei
+  [SupportedChainId.MAINNET]: new Price({
+    baseAmount: ONE_VCOW,
+    quoteAmount: CurrencyAmount.fromRawAmount(WETH9_EXTENDED[SupportedChainId.MAINNET], '37500000000000'),
+  }).invert(),
+  // assuming Rinkeby has same price as Mainnet
+  [SupportedChainId.RINKEBY]: new Price({
+    baseAmount: ONE_VCOW,
+    quoteAmount: CurrencyAmount.fromRawAmount(WETH9_EXTENDED[SupportedChainId.RINKEBY], '37500000000000'),
+  }).invert(),
+  // TODO: wild guess, wxDAI is same price as USDC
+  [SupportedChainId.XDAI]: new Price({
+    baseAmount: ONE_VCOW,
+    quoteAmount: CurrencyAmount.fromRawAmount(WETH9_EXTENDED[SupportedChainId.XDAI], '150000000000000000'),
+  }).invert(),
```

Comment:
This could be a very dumb suggestion, but what about instead of calling `.invert()` on the price you don't invert the base and quote amounts?

Reply:
then I'd need to get the inverse prices, laziness my friend

Reply:
I tend to think the price per vCOW in the investing token to be more readable (0.00375 GNO vs 2666.66666 vCOW), but a better option would be to allow inverting it in the UI.

Anyway, not part of this PR, good as is